binnacle/lib/ui/app_drawer.dart

```diff
@@ -52,16 +51,6 @@ Widget providerList(Bloc bloc) {
     },
   );
 }
-Widget _navigationButton(Bloc bloc, BuildContext context) {
-  return MaterialButton(
-    child: Icon(Icons.navigation),
-    onPressed: () {
-      // Navigator.pop(context);
-      //TODO Navigate to the MAP screen
-      return bloc.initNavigation();
-    });
-
-}
 
 Widget modeToggleSwitch(Bloc bloc, ProviderData providerData) {
```

Comment:
[Nit pick] why are you passing in the bloc and provider data? Shouldn't and widget have access to those.

Reply:
This was when Donovan initiated the nav provider stuff. I removed it as I was able to have it all done in the map screen. Further questions would have to be asked to Donovan.